src/features/organizations/utils/generateTreeData.spec.ts

```diff
@@ -0,0 +1,207 @@
+import { generateTreeData } from './generateTreeData';
+import { TreeItemData } from '../rpc/getOrganizations';
+import { describe, expect, it } from '@jest/globals';
+import { ZetkinMembership, ZetkinOrganization } from 'utils/types/zetkin';
+
+interface mockOrganization {
+  id: number;
+  title: string;
+  parent?: mockOrganization;
+}
+
+interface mockMembership {
+  organization: {
+    id: number;
+    title: string;
+  };
+  role: string;
+}
+
+function mockOrganizations(organization: mockOrganization): ZetkinOrganization {
+  const injectedProperties = {
+    avatar_file: null,
+    country: 'SE',
+    email: null,
+    is_active: false,
+    is_open: false,
+    is_public: true,
+    lang: null,
+    parent: null,
+    phone: null,
+    slug: 'slug',
+  };
+
+  return { ...organization, ...injectedProperties };
+}
+
+function mockMemberships(membership: mockMembership): ZetkinMembership {
```

Comment:
This function creates a single membership, so it should probably be called `mockMembership` (singular).